content/en/modules/introduction_to_dMRI/index.md

```diff
@@ -0,0 +1,179 @@
+---
+type: "modules" # DON'T TOUCH THIS ! :)
+
+# Title of your project (we like creative title)
+title: "Introduction to dMRI"
+
+# Your project GitHub repository URL
+github_repo:
+
+# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
+website:
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/PSY6983-2021/project_template), click `manage topics`.
+# Please only lowercase letters
+tags: [DIPY, python, Tractography, BIDS, Tensor]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+
+summary: "This repo includes tutorial for working with dMRI data using DIPY."
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: "dMRI.png"
+---
+<!-- This is an html comment and this won't appear in the rendered page. You are now editing the "content" area, the core of your description. Everything that you can do in markdown is allowed below. We added a couple of comments to guide your through documenting your progress. -->
+
+## Information
+
+The estimated time to complete this training module is 3.30h.
+
+The prerequisites to take this module are:
+ * Installations
+ * [Introduction to dMRI](https://psy6983.brainhackmtl.org/modules/introduction_to_dMRI/) module.
```

Comment:
This URL will fail once merged. Can we use relative path? I would need someone with more understanding of the Hugo framework to check.(@complexbrains ?)

Reply:
This is fixed.